src/value.rs

```diff
@@ -179,6 +179,47 @@ impl<'t, S> Column<'t, S, i64> {
     }
 }
 
+impl<'t, S> Column<'t, S, String> {
+    /// Check if the column starts with the string pattern.
+    ///
+    /// Matches case-sensitive.
+    pub fn starts_with(&self, pattern: impl Into<String>) -> Column<'t, S, bool> {
+        Glob(self, format!("{}*", pattern.into())).into_column()
+    }
+
+    /// Check if the column ends with the string pattern.
+    ///
+    /// Matches case-sensitive.
+    pub fn ends_with(&self, pattern: impl Into<String>) -> Column<'t, S, bool> {
+        Glob(self, format!("*{}", pattern.into())).into_column()
+    }
+
+    /// Check if the column contains the string pattern.
+    ///
+    /// Matches case-sensitive.
+    pub fn contains(&self, pattern: impl Into<String>) -> Column<'t, S, bool> {
+        Glob(self, format!("*{}*", pattern.into())).into_column()
+    }
+
+    /// Check if the column matches the pattern [docs](https://www.sqlite.org/lang_expr.html#like).
+    ///
+    /// As noted in the docs, it is **case-insensitive** for ASCII characters. Other characters are case-sensitive.
+    /// For creating patterns it uses `%` as a wildcard for any sequence of characters and `_` for any single character.
+    /// Special characters should be escaped with `\`.
+    pub fn like(&self, pattern: impl Into<String> + Clone + 't) -> Column<'t, S, bool> {
```

Comment:
Could you make this also accept `impl IntoColumn<'t, S, Typ = String>` like you did for glob?

Reply:
I thought about this. But the API in `sea-query` of the like operation does not accept a `SimpleExpr` as `glob` does. It wants `Into<String>`. I can't convert from `IntoColumn` to string?

Reply:
Ah, looks like there is an issue for it https://github.com/SeaQL/sea-query/issues/464